horace_core/sqw/@gen_sqw_files_job/private/rundata_write_to_sqw_.m

```diff
@@ -44,6 +44,44 @@
 mpi_obj= MPI_State.instance();
 running_mpi = mpi_obj.is_deployed;
 
+% convert NaN values of runid to sensible values
+% if all values are NaN, assign values by file number
+% if some values are NaN, assign values greater than the maximum non-NaN
+% one
+
+% (a) - find largest non-nan run id
+%     - flag existence of any nans
+maxrunid = NaN;
+hasnans = false;
+for ii=1:nfiles
+    runid = run_files{ii}.run_id;
+    if ~isnan(runid)
+        if isnan(maxrunid)
+            maxrunid = runid;
+        else
+            maxrunid = max(maxrunid, runid);
+        end
```

Comment:
```
>> max(NaN, 3)
max(NaN, 3)

ans =

     3

```
So we can instead:
```suggestion
        maxrunid = max(maxrunid, runid);
```

Reply:
this relies on the reader instinctively knowing this behaviour; the present code works so leaving it but commenting on the alternative